packages/manager/src/managers/customTypes/CustomTypesManager.ts

```diff
@@ -167,6 +180,247 @@ export class CustomTypesManager extends BaseManager {
 		};
 	}
 
+	private updateCRCustomType(
+		args: { customType: CRCustomType } & CustomTypeFieldIdChangedMeta,
+	): { customType: CRCustomType; changed: boolean } {
+		const { previousPath, newPath } = args;
+
+		let changed = false;
```

Comment:
I don't understand this new changed things, it was not here before, right?

Does it correspond to a PR review comment? To a bug? Can you explain a bit more.

Reply:
It was not there, and it's not in response to a PR comment; It was an optimization I thought about doing initially, because we were unnecessarily calling hooks and writing to the disk without any changes, which could contribute to a delay in the updates.